# HG changeset patch
# User Adilson Sandoval <adilsonsdev@gmail.com>
# Date 1603453047 -19800
#      Fri Oct 23 17:07:27 2020 +0530
# Node ID ee793771ebe6d3de29d3052db9853c822bed8aed
# Parent  821eafa266c9a35104e25d98da7f6ff26e6db80d
Issue ui#860 - Added donation for 1% of users

diff -r 821eafa266c9 -r ee793771ebe6 8
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/8	Fri Oct 23 17:07:27 2020 +0530
@@ -0,0 +1,32 @@
+inactive = false
+severity = newtab
+
+title.en-US = AdblockplusDonate202010
+message.en-US = AdblockplusDonate202010
+links = https://new.adblockplus.org/update
+
+target = extension=adblockpluschrome extensionVersion>=3.9.5 locales=ar,am,bg,bn,ca,cs,da,el,en,en-CA,en-GB,en-US,es,es-419,et,fa,fi,fil,gu,he,hi,hr,hu,id,it,ja,kn,ko,lt,lv,ml,mr,ms,nl,no,pl,pt-BR,pt-PT,ro,ru,sk,sl,sr,sv,sw,ta,te,th,tr,uk,vi,zh-CN,zh-TW
+target = extension=adblockplusfirefox extensionVersion>=3.9.5 locales=ar,am,bg,bn,ca,cs,da,el,en,en-CA,en-GB,en-US,es,es-419,et,fa,fi,fil,gu,he,hi,hr,hu,id,it,ja,kn,ko,lt,lv,ml,mr,ms,nl,no,pl,pt-BR,pt-PT,ro,ru,sk,sl,sr,sv,sw,ta,te,th,tr,uk,vi,zh-CN,zh-TW
+
+[1]
+sample = 0.01
+title.en-US = AdblockplusDonate202010
+message.en-US = AdblockplusDonate202010
+
+[2]
+sample = 0.14
+
+[3]
+sample = 0.15
+
+[4]
+sample = 0.20
+
+[5]
+sample = 0.20
+
+[6]
+sample = 0.15
+
+[7]
+sample = 0.15
